services-camel/src/test/java/edu/unc/lib/cdr/BinaryMetadataProcessorTest.java

```diff
@@ -89,9 +89,9 @@ public void validTest() throws Exception {
 
         processor.process(exchange);
 
-        verify(message).setHeader("Checksum", checksum);
-        verify(message).setHeader("MimeType", mimetype);
-        verify(message).setHeader("BinaryPath", BINARY_BASE + "61/67/3d/" + checksum);
+        verify(message).setHeader("CdrChecksum", checksum);
```

Comment:
Since there are constants for these now and this is what constants are supposed to protect you from, lets switch this over to using the constants :)